certification/internal/shell/mounted_engine.go

```diff
@@ -27,16 +28,21 @@ func (e *MountedCheckEngine) ExecuteChecks() error {
 	// We want to know the time just for the check itself, so reset checkStartTime
 	checkStartTime := time.Now()
 
-	// run the validation
-	pass, err := containerutil.RunInsideContainerFS(podmanEngine, targetImage, e.Check.Validate)
+	// If the target is a bundle, we need to run the image filesystem mount logic instead
+	// of the container filesystem mount logic.
+	mountFn := containerutil.RunInsideContainerFS
+	if e.TargetIsBundle {
+		mountFn = containerutil.RunInsideImageFS
```

Comment:
Is this distinction even necessary? Is there an advantage to using the container mount over the image mount for non-bundles? If we can get away with just using the image mount everywhere, then I would just switch the existing mount logic to use the image mount.

Reply:
@bcrochet Okay. I'll make the switch and repush. More to come as I have it.

Reply:
@bcrochet Done!